baby-liminal-extension/Makefile

```diff
@@ -9,6 +9,10 @@ lint: ## Check linting rules
 	cargo +nightly fmt --all --check || exit 1
 	cargo +nightly fmt --all --check $(AT_TEST_CONTRACT) || exit 1
 
+clippy: ## Run clippy
```

Comment:
Good idea with Makefile!

Please let us know when  added CI extends over 10 minutes, we'll then split it somehow (so invariant is to not extend a job over 10 minutes please)